bin/blackbox_view

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+#
+# blackbox_edit.sh -- Decrypt a file temporarily for edition, then re-encrypts it again
```

Comment:
I'm unclear how this is different from `blackbox_cat`.  Is this like /usr/bin/view is to /usr/bin/vim?
If so, please update the comment.